include/dali/core/backend_tags.h

```diff
@@ -54,6 +55,32 @@ struct compute_to_storage<ComputeGPU> {
 template<class ComputeBackend>
 using compute_to_storage_t = typename compute_to_storage<ComputeBackend>::type;
 
+template <typename MemoryKind>
+struct kind2storage_helper;
+
+template <>
+struct kind2storage_helper<cuda::memory_kind::host> {
+  using type = StorageCPU;
+};
+
+template <>
+struct kind2storage_helper<cuda::memory_kind::pinned> {
+  using type = StorageCPU;
+};
+
+template <>
+struct kind2storage_helper<cuda::memory_kind::device> {
+  using type = StorageGPU;
+};
+
+template <>
+struct kind2storage_helper<cuda::memory_kind::managed> {
+  using type = StorageUnified;
+};
+
+template <typename MemoryKind>
+using kind2storage = typename kind2storage_helper<MemoryKind>::type;
+
```

Comment:
I believe it would be better to follow the pattern already present in this file regarding naming, i.e.
```c++
using compute_to_storage_t = typename compute_to_storage<ComputeBackend>::type;
using kind_to_storage_t = typename kind_to_storage<MemoryKind>::type;
```

Reply:
It's not used much, so I won't sweat about those 2 extra characters.

Reply:
Done.

Reply:
It is still the same. Maybe you forgot to commit that file.

Reply:
Nope, it has the trailing `_t` for the `typedef ...::type` - and regarding the x2y, we do have `type2id` - and actually, `id2type` was used without trailing `_t`, so it's not like there's no precedent.

Reply:
I'm not saying we don't. It would be just nice to have aligned the names with `compute_to_storage` in the preceding lines in this file.